client/root.js

```diff
@@ -50,12 +51,23 @@ async function handleLoggedIn( context ) {
 	const state = context.store.getState();
 	const siteSlug = getSiteSlug( state, primarySiteId );
 	const isCustomerHomeEnabled = canCurrentUserUseCustomerHome( state, primarySiteId );
+	const launchpadTasksComplete = false;
+	const launchpadFlow = 'newsletter';
```

Comment:
To confirm, I'd think we'd need to wait to merge this PR to trunk until we have the appropriate conditional logic here?

Reply:
> Just to confirm, I'd think we'd need to wait to merge this PR to trunk until we have the appropriate conditional logic here?

correct

Reply:
> I assume we'll get these values from saved options, and I'm curious how we fetch/access those values on the frontend here. Like maybe we will need to add a method or hook that looks for these options in redux, if so returns them; and if not fetches them from the back end, hydrates the react store, and then returns them.

I updated the sites and site endpoint to include `launchpad_screen` in the site options returned by the endpoint. D86659-code. `site_intent` is already returned from the endpoint.

That should be everything we need to handle this PR.